### PR TITLE
Setup controller

### PR DESCRIPTION
**Pull request by samiullahbahadur**
### **Project requirements**
_**I did the below tasks:**_

-  Create a new app. Give it a meaningful name.
-  Initialize your project with Git.
-  Important: do NOT use rails generate scaffold command. The point of this project is to show you
- how to set up controllers by yourself.
-  Make sure that your project has Postgres database set up.

### CODE REVIEW

File: app/controllers/user_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UserController < ApplicationController
+  def index
+  end
+  def show; end
+end
```
Comment (ShoiraTa) [inline app/controllers/user_controller.rb]: - [ ] You have done a great job properly creating a Users controller, but you also have to create posts_controller with #index and #show actions for this milestone.
Kindly add posts_controller with #index and #show actions (same as users_controller)

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  root "user#index"
+
+  resources :user, only: [:index, :show] do
+  end
```
Comment (ShoiraTa) [inline config/routes.rb]: - [ ] After you create a posts controller, kindly add posts routes nested inside users routes as below: 

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

File: README.md [hunk 3]
```diff
@@ -1 +1,24 @@
-# blog_app
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
```
Comment (ShoiraTa) [inline README.md]: - [ ] Kindly update the readme file 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @samiullahbahadur ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- Default db is Postgres ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/user_controller.rb (hunk 1 above)
Comment (samiullahbahadur) [reply app/controllers/user_controller.rb]: Hi @ShoiraTa 
I have created post controller

File: config/routes.rb (hunk 2 above)
Comment (samiullahbahadur) [reply config/routes.rb]: I did add post root

File: README.md (hunk 3 above)
Comment (samiullahbahadur) [reply README.md]: I did update readme file

File: app/controllers/user_controller.rb (hunk 1 above)
Comment (sja-thedude) [reply app/controllers/user_controller.rb]: Implemented 👍

File: config/routes.rb (hunk 2 above)
Comment (sja-thedude) [reply config/routes.rb]: Implemented 👍

File: README.md (hunk 3 above)
Comment (sja-thedude) [reply README.md]: Implemented 👍

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @samiullahbahadur, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**